openedx/core/djangoapps/zendesk_proxy/utils.py

```diff
@@ -73,7 +93,49 @@ def _std_error_message(details, payload):
                     payload
                 )
             )
+        if additional_info:
+            ticket = json.loads(response.text)['ticket']
```

Comment:
Does there need to be error handling here?

Nit: the requests library provides a `.json()` method on responses.